docs/contributing.md

```diff
@@ -49,10 +49,9 @@ Because of this, setting up and running the tests should be very simple.
 You'll need the following prerequisites:
 
 - Any Python version between **Python 3.9 and 3.12**
-- **virtualenv** or other virtual environment tool
+- [**uv**] (https://docs.astral.sh/uv/getting-started/installation/) or other virtual environment tool
```

Comment:
We should probably run through getting started with the project with this diff - could you try that from scratch just to confirm things run as expected? Thanks!

Reply:
Tested those from scratch and working.